### ISSUE TITLE
Mini hd media box

### ISSUE THREAD

**Issue by Daimyollc** [irplus-remote/irplus-codes.github.io#251]
Lost the remote and the box has no brand. From ATK TECHNOLOGY in China.

**Comment by irplus-remote** [comment]
Hi do you have a link to this device?

**Comment by Daimyollc** [comment]
No,  I do not. However, I did find it the ATK technology created it and the
tkoofn works partially. I can select, move up and down and also turn the
volume up and down.  I just don't know how to exit a movie so I have to
unplug the power and start from the top menu.
On Jun 2, 2016 3:03 AM, "irplus-remote" <notifications@github.com> wrote:

Hi do you have a link to this device?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/irplus-remote/irplus-codes.github.io/issues/251#issuecomment-223249021>,
or mute the thread
<https://github.com/notifications/unsubscribe/AK5-OdTfpDQ-g0F3YlSmMFWtGS7MxKPyks5qHqpsgaJpZM4IsC28>
.

**Comment by irplus-remote** [comment]
Hi, for your information. The Tkoofn codes are the default NEC codes in the 0x00ff range. 

If you go to Example , NEC1-Full  Codeset, you will get the possible 255 combinations for this range. Using this you can find all correct codes
